src/main/java/com/suse/saltstack/netapi/utils/LogUtils.java

```diff
@@ -0,0 +1,297 @@
+    private static boolean isDebugEnabled = false;
+    private static boolean isInfoEnabled = false;
+    private static boolean isWarnEnabled = false;
+    private static boolean isErrorEnabled = false;
+
+    /**
+     * Get logger instance for the caller class
+     * @return {@link Logger}
+     */
+    public static final Logger getLogger() {
+
+        return getLoggerInstance(getCallerClassName());
+    }
+
+    /**
+     * Prints debug message by given logger<br/>
+     * Caller method name will be appended into starting of message
+     * 
+     * @param logger {@link Logger} instance
+     * @param message {@link String} message
+     */
+    protected static final void debug(Logger logger, String message) {
+
+        if (isDebugEnabled) {
+            logger.debug(getCallerMethodName() + "() " + message);
+        }
+    }
+
+    /**
+     * Prints trace message by given logger<br/>
+     * Caller method name will be appended into starting of message
+     * 
+     * @param logger {@link Logger} instance
+     * @param message {@link String} message
+     */
+    protected static final void trace(Logger logger, String message) {
+
+        if (isTraceEnabled) {
+            logger.trace(getCallerMethodName() + "() " + message);
+        }
+    }
+
+    /**
+     * Prints warn message by given logger<br/>
+     * Caller method name will be appended into starting of message
+     * 
+     * @param logger {@link Logger} instance
+     * @param message {@link String} message
+     */
+    protected static final void warn(Logger logger, String message) {
+
+        if (isWarnEnabled) {
+            logger.warn(getCallerMethodName() + "() " + message);
+        }
+    }
+
+    /**
+     * Prints info message by given logger<br/>
+     * Caller method name will be appended into starting of message
+     * 
+     * @param logger {@link Logger} instance
+     * @param message {@link String} message
+     */
+    protected static final void info(Logger logger, String message) {
+
+        if (isInfoEnabled) {
+            logger.info(getCallerMethodName() + "() " + message);
+        }
+    }
+
+    /**
+     * Prints error message by given logger<br/>
+     * Caller method name will be appended into starting of message
+     * 
+     * @param logger {@link Logger} instance
+     * @param message {@link String} message
+     */
+    protected static final void error(Logger logger, String message) {
+
+        if (isErrorEnabled) {
+            logger.error(getCallerMethodName() + "() " + message);
+        }
+    }
+
+    /**
+     * Prints debug message by given logger<br/>
+     * Caller method name will be appended into starting of message
+     * 
+     * @param logger {@link Logger} instance
+     * @param message {@link String} message
+     * @param throwable {@link Throwable} throwable
+     */
+    protected static final void error(Logger logger, String message, Throwable throwable) {
+
```

Comment:
It is because of my eclipse settings, I think I didnt have a proper configuration. I am waiting response if PR will be closed. If not I will commit required changes